examples/alpine-iso.yaml

```diff
@@ -0,0 +1,21 @@
+# This template requires Lima v0.7.0 or later.
+# Using the Alpine 3.20 aarch64 image with vmType=vz requires macOS Ventura 13.3 or later.
```

Comment:
We can probably delete this comment now.

Reply:
Why?

Reply:
I guess I was confused; I thought we required 13.3 for VZ on aarch64 for all images, but it looks like it is Alpine specific, so the comment should stay.